modules/get_contigs_info.nf

```diff
@@ -25,6 +25,6 @@ process GET_CONTIGS_INFO {
     }
     {l+=length(\$0)}
     END {
-    print header[1], l}' $contigsPath | tr -d ">" | sed "1d"
+    print header[1], l}' $contigsPath | tr -d ">" | sed "1d" > output.get_contigs_info
   """
 }
```

Comment:
This is sequence ID and length?
Suggestion to use something existing e.g. seqtk just in case we end up with any weird scenarios in public contigs or long-read assemblies 
https://bioinf.shenwei.me/seqkit/usage/#fx2tab-tab2fx 
https://nf-co.re/modules/seqkit_fx2tab

Reply:
Yes this is it, I'll check to use seqkit then.